astronomer/providers/amazon/aws/example_dags/example_emr_job_flow.py

```diff
@@ -0,0 +1,82 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models.baseoperator import chain
+from airflow.providers.amazon.aws.operators.emr import (
+    EmrCreateJobFlowOperator,
+    EmrTerminateJobFlowOperator,
+)
+
+from astronomer.providers.amazon.aws.sensors.emr import EmrJobFlowSensorAsync
+
+JOB_FLOW_ROLE = os.environ.get("EMR_JOB_FLOW_ROLE", "EMR_EC2_DefaultRole")
+SERVICE_ROLE = os.environ.get("EMR_SERVICE_ROLE", "EMR_DefaultRole")
+AWS_CONN_ID = os.environ.get("ASTRO_AWS_CONN_ID", "aws_default")
```

Comment:
[Airflow documentation](https://airflow.apache.org/docs/apache-airflow/1.10.12/_api/airflow/contrib/operators/emr_create_job_flow_operator/index.html) says the default value of connection_id is either `aws_conn_id='s3_default'` or `emr_conn_id='emr_default'`

Reply:
look like the docs are of an old version. [aws_default](https://github.com/apache/airflow/blob/46c61ed3e94de1484f5f8f3f2fa74e522d67ba9d/airflow/providers/amazon/aws/operators/emr.py#L291) should be fine

Reply:
I verified the code it uses `aws_conn_id= aws_default  ` I guess its old doc, can you please confirm it again